samples/react-chat/src/App.tsx

```diff
@@ -25,31 +26,39 @@ function App() {
     setShowNewChat(false);
   }, []);
 
+  const [usePremiumApis, setUsePremiumApis] = useState<boolean>(false);
+  const onToggleChanged = useCallback((ev: ChangeEvent<HTMLInputElement>, data: SwitchOnChangeData) => {
+    setUsePremiumApis(data.checked ?? false);
+  }, []);
+
   return (
     <div className="App">
-      <header className="App-header">
-        Mgt Chat test harness
-        <br />
-        <Login />
-      </header>
-      <main className="main">
-        <div className="chat-selector">
-          <ChatList chatSelected={chatSelected} />
-          Selected chat: {chatId}
-          <br />
-          <button onClick={() => setChatId('')}>Clear selected chat</button>
+      <FluentProvider theme={teamsLightTheme}>
+        <header className="App-header">
+          Mgt Chat test harness
           <br />
-          <button onClick={() => setShowNewChat(true)}>New Chat</button>
-          {showNewChat && (
-            <div className="new-chat">
-              <NewChat onChatCreated={onChatCreated} onCancelClicked={() => setShowNewChat(false)} mode="auto" />
-            </div>
-          )}
-        </div>
+          <Login />
+        </header>
+        <main className="main">
+          <div className="chat-selector">
+            <Switch onChange={onToggleChanged} checked={usePremiumApis} label="Use premium APIs?" />
```

Comment:
Add the same to contoso app

Reply:
I'm not sure about that TBH. I put it here to allow for us to test this functionality, it's not really something we'd want end users in control of in the real world as it depends on the config of the Entra App Registration to provide billing support.